packages/map/src/index.ts

```diff
@@ -137,53 +149,63 @@ export class ReactiveWeakMap<K extends object, V> extends WeakMap<K, V> {
   #keyTriggers = new TriggerCache<K>(WeakMap);
   #valueTriggers = new TriggerCache<K>(WeakMap);
 
-  constructor(initial?: Iterable<readonly [K, V]> | null) {
+  constructor(entries?: readonly [K, V][] | null) {
     super();
-    if (initial) for (const v of initial) super.set(v[0], v[1]);
+    if (entries) for (const entry of entries) super.set(...entry);
```

Comment:
I should probably use `apply` here to avoid destructering

Reply:
Why can't we use `super(entries)`?

Reply:
we call `super.set` here not `super`. We want to call set for each entry not initialize a map for each entry :) Hope that explains.